### PR TITLE
added view call method, test and example

### PR DESCRIPTION
**Pull request by victormb84**

### CODE REVIEW

**Comment by victormb84** [comment]
Added test, example and method to view a Call

File: messagebird/call_data.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from messagebird.base import Base
+
+class CallData(Base):
+    
+    def __init__(self):
```
Comment (dysosmus) [inline messagebird/call_data.py]: You also have `webhook` which is composed of a `token` and a `url`

Comment (victormb84) [reply messagebird/call_data.py]: ok, i will add that

Comment (victormb84) [reply messagebird/call_data.py]: @dysosmus just added webhook to Call class

**Pull request merged**